internal/controller/indexer/indexer.go

```diff
@@ -54,6 +54,10 @@ const (
 	ControllerName            = "controllerName"
 )
 
+const (
+	ingressClassNameAnnotation = "kubernetes.io/ingress.class"
```

Comment:
Is it more appropriate to allow cross package use of this constant?

Reply:
I am going to adjust the code, as the current code is a bit redundant.

Reply:
updated.